idseq_pipeline/commands/host_filtering_functions.py

```diff
@@ -821,25 +808,11 @@ def truncate_input(fastq_file):
         assert t.completed and not t.exception
 
     # Downloaded files here are unzipped but not yet truncated.
```

Comment:
Looks like this comment is no longer accurate after your change.   Perhaps remove the comment.